### PR TITLE
add script of delete unnecessary test source.#41

### PR DESCRIPTION
**Pull request by btnoguchimnm**
please review. #41

### CODE REVIEW

File: first-spring-security/scripts/generate-security-project.sh [hunk 1]
```diff
@@ -109,4 +109,7 @@ cp -p -r ./common-test/resources ./target-project/first-spring-security-mybatis3
 
 cp -p ./first-spring-security/src/test/java/com/example/security/selenium/FirstSpringSecurityTest.java ./target-project/first-spring-security-mybatis3-multi/first-spring-security-mybatis3-multi-selenium/src/test/java/com/example/security/selenium/security/
 cp -f -p -r ./common-test/java/selenium ./target-project/first-spring-security-mybatis3-multi/first-spring-security-mybatis3-multi-selenium/src/test/java/com/example/security/
-cp -f -p -r ./common-test/resources ./target-project/first-spring-security-mybatis3-multi/first-spring-security-mybatis3-multi-selenium/src/test
\ No newline at end of file
+cp -f -p -r ./common-test/resources ./target-project/first-spring-security-mybatis3-multi/first-spring-security-mybatis3-multi-selenium/src/test
+
+# delete test sources
```
Comment (btnoguchimnm) [inline first-spring-security/scripts/generate-security-project.sh]: [3794a5e]
I added reason to remove test source in comments.

File: first-spring-security/scripts/generate-security-project.sh [hunk 2]
```diff
@@ -109,4 +109,7 @@ cp -p -r ./common-test/resources ./target-project/first-spring-security-mybatis3
...
+rm -rf ./target-project/first-spring-security-mybatis3-multi/first-spring-security-mybatis3-multi-selenium/src/test/java/com/example/security/selenium/welcome/
```
Comment (btnoguchimnm) [inline first-spring-security/scripts/generate-security-project.sh]: [c0cca37 ]
I selected the latter and fix it.

File: first-spring-security/scripts/generate-security-project.sh (hunk 1 above)
Comment (btyoshikawaa) [inline first-spring-security/scripts/generate-security-project.sh]: Uh...
I think it is enough as follows. 
`delete unuse (or conflict) test sources in blank project`

Comment (btnoguchimnm) [inline first-spring-security/scripts/generate-security-project.sh]: [8a0b600 ]
Sorry, I explained too much.
I corrected it to what I was presented.

**Comment by btyoshikawaa** [comment]
@btnoguchimnm 
OK. But stay to leave this the time of travis busy.

**Pull request merged**